.gitignore

```diff
@@ -25,3 +25,4 @@ features/**/src/main/feature
 .factorypath
 pom.xml.versionsBackup
 .openapi-generator
+bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/testdevice.xml
```

Comment:
Can this ignore be removed? This file seems to be part of the PR.

Reply:
this is just a testdevice. At the moment it's for testing/debuging new devices wich are used by some other users.
It will be removed later

Reply:
I was asking because of this unneccessary change in your PR, the build servers will rebuild of all add-ons when it builds this PR, instead of only the tapocontrol add-on. So instead of having feedback in a couple of minutes it takes hours. :wink:

Reply:
I will remove it with the next git-push. I'll create a new branch for test pruposes

Reply:
Did you push your changes? It's still there and so the testDevice.xml.

Reply:
Damn forgot to delete this....

Reply:
Now it shoulb be cleared :)